src/icons/index.js

```diff
@@ -5,6 +5,7 @@ export atom from './atom.svg';
 export bower from './bower.svg';
 export bq from './bq.svg';
 export clojure from './clojure.svg';
+export database from './database.svg';
```

Comment:
This file is sorted alphabetically. Can you put this `export` after the other `export`s that start with "c"?

Reply:
Done this.